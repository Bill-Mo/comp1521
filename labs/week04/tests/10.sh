echo 1 | /home/dp1092/bin/spim -file negative.s | sed -e '1d;s/n  = //'
