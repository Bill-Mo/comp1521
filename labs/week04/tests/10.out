Enter a number: You have entered a positive number.
